pkg/apis/config/features.go

```diff
@@ -40,10 +40,11 @@ const (
 
 func defaultFeaturesConfig() *Features {
 	return &Features{
-		MultiContainer:       Disabled,
-		PodSpecFieldRef:      Disabled,
-		PodSpecDryRun:        Allowed,
-		ResponsiveRevisionGC: Disabled,
+		MultiContainer:                 Disabled,
+		PodSpecFieldRef:                Disabled,
+		PodSpecDryRun:                  Allowed,
+		PreventActiveRevisionDeletions: Disabled,
```

Comment:
did you mean for the config map key to be singular but the field to be plural?

Reply:
Thanks. Fixed.